modin/core/io/file_dispatcher.py

```diff
@@ -111,7 +111,7 @@ def __exit__(self, *args):
         self.file.close()
 
 
-class FileDispatcher:
+class FileDispatcher(LoggerBase):
```

Comment:
Will this still work? The metaclass didn't work for the classmethods in the previous implementation.

Reply:
Manually ran a few `read_csv` and `read_excel` commands, which seem to print out the dispatcher class logs correctly.